app/controllers/gobierto_admin/gobierto_calendars/calendar_configuration_controller.rb

```diff
@@ -33,8 +34,32 @@ def update
         end
       end
 
+      def sync_calendars
+        @calendar_configuration_form = CalendarConfigurationForm.new(current_site: current_site, collection_id: @collection.id)
+        if (calendar_integration = @calendar_configuration_form.calendar_integration_class).present?
+          calendar_integration.sync_person_events(@calendar_configuration_form.collection_container)
+          publish_calendar_sync_activity(@calendar_configuration_form)
+        end
+        redirect_to edit_admin_calendars_configuration_path(@collection)
```

Comment:
Could you add a flash message after the sync? Right now it works but there's no clear feedback for the user that it worked